src/main/java/com/locat/api/domain/endpoint/controller/EndpointController.java

```diff
@@ -0,0 +1,50 @@
+package com.locat.api.domain.endpoint.controller;
+
+import com.locat.api.domain.endpoint.dto.EndpointRegistrationRequest;
+import com.locat.api.domain.endpoint.service.PlatformEndpointService;
+import com.locat.api.domain.user.entity.User;
+import com.locat.api.domain.user.entity.UserEndpoint;
+import com.locat.api.domain.user.service.UserEndpointService;
+import com.locat.api.domain.user.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("v1/endpoint")
+public class EndpointController {
+
+    private final PlatformEndpointService platformEndpointService;
+    private final UserEndpointService userEndpointService;
+    private final UserService userService;
+
+    @PostMapping("/register")
+    public ResponseEntity<Void> register(
+            @RequestBody final EndpointRegistrationRequest request) {
```

Comment:
Request Body를 `jakarta.validation` 어노테이션으로 검증하려면 컨트롤러 해당 부분에 `@Valid` 해주셔야 할 것 같아요!
```suggestion
            @RequestBody @Valid final EndpointRegistrationRequest request) {
```